source/jsonnet/northern-ireland/individual/blocks/identity-and-health/childhood_religion.jsonnet

```diff
@@ -63,11 +63,11 @@ local question(title, otherDescription) = {
   page_title: 'Childhood religion',
   question_variants: [
     {
-      question: question(nonProxyTitle, 'You can enter your religion on the next question'),
+      question: question(nonProxyTitle, 'You can enter the religion on the next question'),
       when: [rules.isNotProxy],
     },
     {
-      question: question(proxyTitle, 'You can enter their religion on the next question'),
+      question: question(proxyTitle, 'You can enter the religion on the next question'),
```

Comment:
As there is now no difference between the proxy and non-proxy descriptions, the description no longer needs to be passed as an argument to `question`.

Reply:
Removed, thanks!👍